.github/actions/build-dotnet/Build-DotNetSolution.ps1

```diff
@@ -50,6 +48,11 @@ if (Test-Path src/Utilities/Lombiq.Gulp.Extensions/Lombiq.Gulp.Extensions.csproj
     Write-Output "::endgroup::"
 }
 
+# This prepares the solution with the Lombiq.Analyzers files. The output and exit code are discarded because they will
+# be in error if there is a project without the LombiqNetAnalyzers target. Then there is nothing to do, and the target 
+# will still run on the projects that have it.
+dotnet msbuild '-target:Restore;LombiqNetAnalyzers' $Solution | Out-Null || bash -c 'true'
```

Comment:
So here's my confusion: In the [Build.props](https://github.com/Lombiq/.NET-Analyzers/pull/57#discussion_r935460867) you write that we need to run `dotnet msbuild -target:LombiqNetAnalyzers` but here we also call the `Restore` target. Why?

Reply:
> [If you are using NuGet, then the argument should be "-target:Restore;LombiqNetAnalyzers" instead (include the quotes) to restore all package dependencies first.](https://github.com/Lombiq/.NET-Analyzers/blob/101ccada871b31c8896d87f3c296e4ca87e7ddc3/Lombiq.Analyzers/Build.props#L78-L79)